commit-branch.md

```diff
@@ -176,30 +176,50 @@ No tak to už umíme, tak si pojďme ten soubor commitnout příkazem:
 
 ```
 $ git add --all
+```
+
+**GOTCHA:** defaultní editor pro úpravu commit messages a vůbec všeho textového v Gitu je VIM.
+A skoro nikdo neví, jak do něj něco napsat nebo hůř, jak se z něj dostat, takže nastavíme náš oblíbený editor - třeba Nano:
+```
+git config --global core.editor nano
+```
 
+A provedeme commit:
+```
 $ git commit
 ```
-**GOTCHA:** defaultní editor pro úpravu commit messages a vůbec všeho textového v Gitu je VIM.
-A nikdo neví, jak do něj něco napsat nebo hůř, jak se z něj dostat proto:
+
+Nyní napíšeme nějakou commit message - to, co jsme udělali. V našem případě asi něco jako:
+```
+přidání souboru pro poznámky notes.md
+```
+A uložíme
+
+```
+Ctrl + o
```

Comment:
nechal bych tu poznámky, jak to dělat ve VIMu to proto, že většina lidí se potká s VIMem, pokud dělají Git prvně.